packages/ListBoxWithTags/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+- Add aria and accessibility to trigger component for screen reader
+- Added uuid package
```

Comment:
This is an internal implementation detail that I don't think needs to be logged.  🤷‍♂️